test/integration/organisation_test.rb

```diff
@@ -178,10 +221,32 @@ class OrganisationHeaderTest < ActionDispatch::IntegrationTest
       }
     }
 
+    @content_item_blank = {
```

Comment:
Not sure this belongs in this commit, especially as other tests are added separately.

Reply:
Which commit is this in?